docs/FAQ.md

```diff
@@ -33,6 +33,29 @@ revision visible again.
 
 See [revsets] and [templates] for further guidance.
 
+### `jj` is said to record the working after `jj log` and every other command. Where can I see these automatic "saves"?  
+
+Indeed, every `jj` command updates the current "working-copy" revision, marked 
+with `@` in `jj log`. You can notice this by how the [commit id] of the
+working copy revision changes when it's updated. Note that, unless you move to
+another revision (with `jj new` or `jj edit`, for example), the [change id] will 
+not change.
+
+If you expected to see a historical view of your working-copy changes in 
+`jj log`, as a chain in a parent-child relationship, this is not the case. 
+Instead each commit gets amended and the commit-id changes.
+
+You can see the history of these changes using `jj obslog`. This will show the 
+history of the commits that were previously the "working-copy commit", since 
+the last time the change id of the working copy commit changed. The obsolete 
+changes will be marked as "hidden". They are still accessible with any `jj` 
+command (`jj diff`, for example), but you will need to use the commit id to 
+refer to hidden commits.
+
+You can also use `jj obslog -r` on revisions that were previously the 
+"working-copy" revisions. Use `jj interdiff` as an easy way to see a commit's 
+evolution.
```

Comment:
I think `jj obslog -p` is a better suggestion since it's less manual (I'm guessing you meant to pass two consecutive commits to `jj interdiff`, but that's basically what `jj obslog -p` does).

Reply:
done.